src/Illuminate/Container/Container.php

```diff
@@ -285,11 +286,17 @@ public function bindMethod($method, $callback)
      *
      * @param  array|string $method
      * @return string
+     *
+     * @throws \InvalidArgumentException
      */
     protected function parseBindMethod($method)
     {
         if (is_array($method)) {
-            return $method[0].'@'.$method[1];
+            if (($count = count($method)) > 1) {
+                return $method[0].'@'.$method[1];
+            }
+
+            throw new InvalidArgumentException("Method should be array with 2 parameters, {$count} given");
```

Comment:
Arrays don't have "parameters". They have keys and values, and a length.

Reply:
Changed, anyway...

Could you help me with correct message?